File: Labs/lab06/whale.pl
#!/usr/bin/perl -w

use strict;

my $whale = $ARGV[0];
my $podCount = 0;
my $whaleCount = 0;

foreach my $line (<STDIN>) {
	chomp $line;
	if ($line =~ /$whale/) {
		$podCount += 1;

		my @count = split / /, $line;

		foreach my $counter (@count) {
			if ($counter =~ /[0-9]/) {
				$whaleCount += "$counter";
			}
		}
	}
}
print "$whale observations: $podCount pods, $whaleCount individuals\n";

File: Labs/lab06/whale.py
#!/usr/bin/python

import sys

whale = sys.argv[1]

pods = 0
individuals = 0

for line in sys.stdin:

	words = line.split(" ",1)

	if (words[1] == (whale + '\n')):
		pods += 1
		individuals += int(words[0])

print (whale + " observations: %d pods, %d individuals"%(pods, individuals))